generators/app/package-json.js

```diff
@@ -142,7 +143,12 @@ const packageJSON = (current, context) => {
       'clean': 'shx rm -rf ./dist ./test/dist',
       'postclean': 'shx mkdir -p ./dist ./test/dist',
       'lint': 'vjsstandard',
-      'prepublish': 'not-in-install && npm run build && pkg-ok || in-install',
+      'prepublish': 'not-in-install && npm run verify || in-install',
+      'preverify': 'npm run build',
```

Comment:
running the build as a preverify seems weird. I think it would be better to keep it in prepublish as a serial run via npm-run-all `run-s build verify`.

Reply:
Well in order to run verify manually (if anyone ever does that). we will have to have build happen first. That is why I did a `preverify`

Reply:
Right, though, I would still find it confusing and weird that prepublish doesn't call build directly.
Maybe it should be `run-s build verify:*`? That way, it doesn't run the build again.